packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -299,8 +317,14 @@ class NewDisk extends Component {
     resourceSet => get(resourceSet, 'limits.disk.available')
   )
 
+  _isSrOnSameHost = createSelector(
```

Comment:
Naming: either `NewDisk` is aware that "illegal" means "not on same host", either it's not, but don't mix both.